controllers.md

```diff
@@ -48,27 +48,27 @@ Below is an example of a basic controller class. Note that the controller extend
         }
     }
 
-You can define a route to this controller action like so:
+你可以像這樣定義一個對應到此控制器動作的路由：
 
     Route::get('user/{id}', 'UserController@show');
 
-Now, when a request matches the specified route URI, the `show` method on the `UserController` class will be executed. Of course, the route parameters will also be passed to the method.
+現在，當請求配對到指定的路由 URI 時，`UserController` 類別的 `show` 方法將被執行。當然，路由參數也會傳遞給方法。
 
-> {tip} Controllers are not **required** to extend a base class. However, you will not have access to convenience features such as the `middleware`, `validate`, and `dispatch` methods.
+> {提示} 控制器並非 **一定** 得繼承基礎類別。 但是，你將無法存取一些方便的功能，像是 `中介層(middleware)`、`驗證(validate)` 以及 `任務派發(dispatch)` 之類。
 
 <a name="controllers-and-namespaces"></a>
-### Controllers & Namespaces
+### 控制器與命名空間
 
-It is very important to note that we did not need to specify the full controller namespace when defining the controller route. Since the `RouteServiceProvider` loads your route files within a route group that contains the namespace, we only specified the portion of the class name that comes after the `App\Http\Controllers` portion of the namespace.
+有一點非常重要，那就是我們在定義控制器路由時，不需要指定完整的控制器命名空間。當 `RouteServiceProvider` 載入路由檔案時，其中有包含了命名空間的路由群組，所以只需要指定命名空間 `App\Http\Controllers` 之後部分的類別名稱即可。
```

Comment:
有一點非常重要，當我們在定義控制器路由時，不需要指定完整的控制器命名空間。

這邊的 `Since` 應該不是翻譯為**當**，用**因為**比較恰當。
因為 `RouteServiceProvider` 載入你的路由檔案已包含了路由群組的命名空間，我們只需要指定 `App\Http\Controllers` 後的部份命名空間類別名稱。